lm_eval/api/model.py

```diff
@@ -1,8 +1,13 @@
 import abc
+import os
 
 from typing import Union
+from sqlitedict import SqliteDict
```

Comment:
I'm not familiar with `sqlitedict`. Do we need to add this as part of the required libraries?

Reply:
ah, if it’s not currently a requirement then yep, should be a default one , can do